docs/DeveloperGuide.md

```diff
@@ -263,18 +264,22 @@ The activity diagram shows the general sequence of steps when a user interacts w
 <br>
 
 ### Clear person feature
-
-#### Implementation
-
 #### Design considerations
+**Aspect: Should `clear persons` also delete appointments**
+- Alternative 1 (Current choice): The `clear persons` command should also clear the appointments else there will be alot of appointments inside the application that are linked to person ids that don't exists. Causing confusion for the users.
+- Alternative 2 : We assume the user will run `clear appt` command after they run `clear persons` to remove the previous appointments
 
 <br>
 
 ### Add appointment feature
-
-#### Implementation
-
 #### Design considerations
+**Aspect: Should we implement as `addAppt` or `add appt`**
+- Alternative 1 (Current choice): Implement the add appointment feature as `add appt`
+  - Pros: Allows us to use the existing infrastructure, just have to add code to detect weather the entity is `appt` or not.
```

Comment:
Minor spelling error, should be "whether"

Reply:
Do you think this aspect can be generalised for the other commands as well, since they also check for the entity type?